Managers/SceneMan.h

```diff
@@ -1179,6 +1179,19 @@ class SceneMan : public Singleton<SceneMan>, public Serializable {
     float CastMaxStrengthRay(const Vector &start, const Vector &end, int skip, unsigned char ignoreMaterial);
     float CastMaxStrengthRay(const Vector &start, const Vector &end, int skip) { return CastMaxStrengthRay(start, end, skip, g_MaterialDoor); };
 
+//////////////////////////////////////////////////////////////////////////////////////////
+// Method:          CastMaxStrengthRayMaterial
+//////////////////////////////////////////////////////////////////////////////////////////
+// Description:     Traces along a vector and returns the strongest of all encountered pixels' materials
+//                  This will take wrapping into account.
+// Arguments:       The starting position.
+//                  The ending position.
+//                  For every pixel checked along the line, how many to skip between them
+//                  for optimization reasons. 0 = every pixel is checked.
+//                  A material ID to ignore, IN ADDITION to Air. This defaults to doors, for legacy script purposes
+// Return value:    The strongest material encountered
+    const Material * CastMaxStrengthRayMaterial(const Vector &start, const Vector &end, int skip, unsigned char ignoreMaterial);
```

Comment:
Do we need the legacy purpose accounting for doors, if this isn't a legacy method?
Though I guess you kept it cause this is kinda just a reskin of CastMaxStrengthRay